app/build.gradle

```diff
@@ -56,6 +58,7 @@ android {
         versionCode 19
         versionName "1.1.0"
         buildConfigField "boolean", "DEBUG_MODE", "true"
+        buildConfigField "boolean", "CRASH_REPORTING_ENABLED", "true"
```

Comment:
I don't get the default logic here. Should it be enabled for **debugAlpha** and **release** and disabled for the rest ?
If so, then it should be false here and true down below in **debugAlpha**

Reply:
this flag mean that option to ENABLE it in settings is visible. If this is false, you don't even have the possibility to turn on crash reporting in the settings, like for debug build. Maybe I should rename to something like `CRASH_REPORTING_AVAILABLE` ?

Reply:
I agree `CRASH_REPORTING_AVAILABLE` seems more reasonable.

Reply:
Yes 💯